pennylane/qnn/torch.py

```diff
@@ -208,7 +219,9 @@ def __init__(self, qnode, weight_shapes: dict, init_method: Optional[Callable] =
         super().__init__()
 
         weight_shapes = {
-            weight: (tuple(size) if isinstance(size, Iterable) else (size,) if size > 1 else ())
+            weight: (
+                tuple(size) if isinstance(size, Iterable) else () if size in [0, 1] else (size,)
```

Comment:
Minor, as per Tom's comment. Can be ignored if preferred.
```suggestion
                tuple(size) if isinstance(size, Iterable) else () if size in [1] else (size,)
```

Reply:
I changed this in commit https://github.com/PennyLaneAI/pennylane/pull/2678/commits/339ab96f7cbbef889361176c1c5aefe3a60c6415. It is important to know that this small change affects the usage of the `TorchLayer` class. Before a user could use `size = 0` or `size = 1` to define a scalar tensor. Now only `size = 1` can be used. Using `size = 0` will create an empty Tensor, which I don't think has any usage.

Reply:
Thank you! I've added a [suggested comment to the changelog](https://github.com/PennyLaneAI/pennylane/pull/2678/files#r894860328) to document this change in behaviour as a breaking change. The entry also mentioned the behaviour change for the definition of weights with negative shapes.

Reply:
Everything good! 👍